Library/Tactical Shooting/Tactical Shooting Skills.skl

```diff
@@ -720,26 +700,6 @@
 				"Tactical Shooting",
 				"Technique"
 			]
-		},
-		{
-			"type": "technique",
-			"id": "f18e8dda-cb21-4413-a443-636f10c67901",
-			"name": "Targeted Attack (Chiks in torso armor)",
-			"reference": "TS45",
-			"difficulty": "h",
-			"points": 2,
-			"limit": -4,
-			"default": {
-				"type": "skill",
-				"name": "Guns",
-				"specialization": "@Guns@",
-				"modifier": -8
```

Comment:
Again, I don't have the relevant books, so can't say for certain, but this appears to be a special sub-case (torso armor vs other armor) with a different modifier. Is that not the case?

Reply:
Yes, but both of them were entered twice. With this change only one copy of each technique is present.